dropletutils-scripts-post-install-tests.bats

```diff
@@ -15,14 +15,35 @@
     [ -f  "$raw_matrix" ]
 }
 
+# create an sdrf file to incorporate into SCE object 
+@test "fetch a dummy sdrf for testing" {
+    if [ "$use_existing_outputs" = 'true' ] && [ -f "$test_sdrf" ]; then
+        skip "$test_sdrf exists and use_existing_outputs is set to 'true'"
+    fi
+
+    run rm -rf $test_sdrf && dropletutils-make-test-sdrf.R -b $barcodes -o $test_sdrf
+    echo "status = ${status}"
+    echo "output = ${output}"
+ 
+    [ "$status" -eq 0 ]
+    [ -f  "$test_sdrf" ]
+}
+
+
 # Create the Matrix object
 
 @test "SingleCellExperiment object creation from 10x" {
```

Comment:
Could we have an additional similar test to this /without/ specification of any of the metadata parameters? I want to be sure that this change does not impact additional functionality.